contrib/test/integration/main.yml

```diff
@@ -114,8 +114,8 @@
       include: "build/kubernetes.yml"
       vars:
           force_clone: True
-          k8s_git_version: "release-1.12"
-          k8s_github_fork: "mrunalp"
+          k8s_git_version: "master"
```

Comment:
Let's stick to a commit, otherwise we'll get hit by issues again (like when upstream added more tests and out ci started timing out). It's our duty to update it from time to time.

Reply:
I think that helps us catch those issues earlier ;)